src/ring_theory/ideals.lean

```diff
@@ -164,6 +165,53 @@ theorem is_coprime_self {x : α} :
   is_coprime x x ↔ is_unit x :=
 by rw [← span_singleton_eq_top]; simp [is_coprime]
 
+/-- This is `is_coprime_def` but specialized to `1`. -/
```

Comment:
Would you mind putting the entire `is_coprime` business outside of the `ideal` namespace? Because `ideal.is_coprime` suggests that this is about coprimality of ideals, instead of elements.

Reply:
done